src/perf/lib/SecNetPerfMain.cpp

```diff
@@ -96,6 +100,49 @@ PrintHelp(
         );
 }
 
+#ifdef QUIC_USE_RAW_DATAPATH
+CXPLAT_THREAD_CALLBACK(ControlThread, Context)
+{
+    WSADATA WsaData;
+    WSAStartup(MAKEWORD(2, 2), &WsaData);
```

Comment:
This all is not cross platform. I want to keep the code cross-platform.

Reply:
I only changed it for XDP codepath which is windows only. When XDP can coexist with any other datapath, we can simply remove this workaround.

Reply:
While XDP support is currently Windows-only, that won't be true forever, and likely not even in the long term; but I guess we might end up merging the XDP and socket datapaths before we add Linux support. But by doing it this way, you sign up for the possibility of having to do this in a cross-platform way later if we add Linux support before combining the datapaths.

Reply:
Yeah, it should be within 5 lines of code change to make this piece of code work for linux if linux support comes sooner than combining datapaths.